internal/announcements/monitor.go

```diff
@@ -0,0 +1,8 @@
+// monitor.go
+
+package announcements
+
+type Monitors interface {
```

Comment:
I would rename the interface to `Monitor` instead of `Monitors` (to the object not the verb)